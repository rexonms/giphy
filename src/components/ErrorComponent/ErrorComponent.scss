.container {
  text-align: center;
  display: flex;
  justify-content: space-between;
  flex-direction: column;
  height: 100vh;
  p {
    color: red;
  }
}
