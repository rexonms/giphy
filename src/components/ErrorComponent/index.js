import React from 'react';

import Styles from './ErrorComponent.scss';

const propTypes = {};
const defaultProps = {};

const ErrorComponent = () => (
  <div className={Styles.container}>
    <p>OMG what happen!</p>
  </div>
);

ErrorComponent.propTypes = propTypes;
ErrorComponent.defaultProps = defaultProps;

export default ErrorComponent;
