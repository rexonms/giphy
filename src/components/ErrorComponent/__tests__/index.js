import React from 'react';
import { shallow } from 'enzyme';

import ErrorComponent from './../index';

describe('Error Component', () => {
  let props;
  let shallowErrorComponent;
  const errorComponent = () => {
    if (!shallowErrorComponent) {
      shallowErrorComponent = shallow(<ErrorComponent {...props} />);
    }
    return shallowErrorComponent;
  };
  beforeEach(() => {
    props = {};
    shallowErrorComponent = undefined;
  });

  it('should render previous snapshot', () => {
    expect(errorComponent()).toMatchSnapshot();
  });
});
