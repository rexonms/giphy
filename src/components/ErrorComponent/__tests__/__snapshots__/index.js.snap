// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Component should render previous snapshot 1`] = `
<div
  className="container"
>
  <p>
    OMG what happen!
  </p>
</div>
`;
